### PR TITLE
Put YamUI use instructions at top in better-developed section

### PR DESCRIPTION
**Pull request by annr**
This is so devs can understand how to use YamUI from the README -- it was buried in the body of the README before.

### CODE REVIEW

**Comment by annr** [comment]
@unindented please review when you get a chance -- no hurry.

File: README.md [hunk 1]
```diff
@@ -2,9 +2,39 @@
...
 
-It is built with [React](https://reactjs.org/) on top of [Fabric](https://dev.office.com/fabric#/components/) components. Unit tests are run through [Jest](https://facebook.github.io/jest/), isolated development environment and documentation is provided by [Storybook](https://github.com/storybooks/storybook), and visual diff regression is done with [BackstopJS](https://garris.github.io/BackstopJS/).
+It is built with [React](https://reactjs.org/) on top of [Office UI Fabric](https://dev.office.com/fabric#/components/) components. Unit tests are run through [Jest](https://facebook.github.io/jest/), isolated development environment and documentation is provided by [Storybook](https://github.com/storybooks/storybook), and visual diff regression is done with [BackstopJS](https://garris.github.io/BackstopJS/). Visual diff tasks run within a [Docker](https://www.docker.com/) container to ensure consistency between local development environments and CI.
 
-Visual diff tasks run within a [Docker](https://www.docker.com/) container to ensure consistency between local development environments and CI.
+## Using YamUI in your own app
+
+If necessary, add YamUI to your project.
+
+```
```
Comment (unindented) [inline README.md]: Here you're missing the code block language. It'd probably be ```sh for shell commands.

Comment (annr) [reply README.md]: I see. And now the purpose of the linter is clearer.

**Comment by annr** [comment]
I didn't manually bump the version. I used the instructions but `npm version patch "Release version %s!"` didn't work. Maybe the bang just needs to be removed.

File: README.md (hunk 1 above)
Comment (annr) [reply README.md]: I could have sworn that the error was the same as previous -- that whitespace was necessary. hrm.

**Comment by unindented** [comment]
You're right, the `!` is a typo, and it breaks the command.

**Comment by unindented** [comment]
I'd remove the `0.0.18` commit and do it through the command. Because this branch doesn't have a tagged commit, it won't trigger the auto-release to npm.

**Comment by annr** [comment]
I manually created a release in GitHub (I did this previously). I'm going to see if this works to bump the version and publish. It's an experiment.

**Comment by unindented** [comment]
Uh, @bryan-gislason had already pushed version `0.0.18` three weeks ago? Did he not commit his change to the `package.json` file?

screen shot 2017-10-19 at 16 10 06

**Pull request merged**